src/components/30-organisms/modal/index.js

```diff
@@ -110,8 +110,11 @@ class AXAModal extends InlineStyles {
   firstUpdated() {
     this.inlineStyles('resetHeadingCss');
 
-    // add eventListener to close modal when pressing outside the modal
-    document.body.addEventListener('click', ev => {
+    // add eventListener to close modal when pressing outside the modal.
```

Comment:
wording: pressing -> clicking

Reply:
It's not my text but i changed it anyway ;)